bool errorOrSafeStart
bool tempLimiter
bool highTargetSpeed
bool lowTargetSpeed
bool accelDeccelLimiter
bool an1Limit
bool an2Limit
bool usbKill
